massgen/tests/test_position_bias_calibration.py

```
@@ -0,0 +1,191 @@
+            agent_mapping=self.mapping,
+            order_seed=2,
+        )
+        assert self._first_label(conv["user_message"]) == "agent3"
+
+
+class TestOwnLastOrderSeed:
+    """compute_own_last_order_seed must place the scoring agent's own answer LAST,
+    derived from the answering subset (not the global roster) so it stays correct
+    when only a non-contiguous subset has answered."""
+
+    mapping = {"agent_a": "agent1", "agent_b": "agent2", "agent_c": "agent3"}
```

Ruff (0.15.13)

[warning] Mutable default value for class attribute

(RUF012)